timesketch/frontend/src/components/Sigma/SigmaList.vue

```diff
@@ -0,0 +1,129 @@
+<!--
+Copyright 2021 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <div>
+    <b-table
+      v-if="sigmaRuleList"
+      :data="sigmaRuleList"
+      :current-page.sync="currentPage"
+      :per-page="perPage"
+      paginated
+      pagination-simple
+      pagination-position="bottom"
+      default-sort-direction="desc"
+      sort-icon="arrow-down"
+      sort-icon-size="is-small"
+      icon-pack="fas"
+      icon-prev="chevron-left"
+      icon-next="chevron-right"
+      default-sort="title"
+    >
+      <b-table-column field="id" label="Rule ID" v-slot="props" sortable searchable>
```

Comment:
I'm about to mock my UI idea here. I'll update tomorrow.
I'm windering if we should just have one button to take the user to the explore view - with the generated query as the parameter?

Reply:
yes that is super easy to do and is already in the overview table, I just did not implement it here to wait on the brainstorming